### DISCUSSION TITLE
Is permissions

### DISCUSSION THREAD

**Discussion by Chara179352468** [BG-Software-LLC/SuperiorSkyblock2#1131] [Denied Feature Requests] [answered]
make sure we can't do that

a visitor can have all the permissions it makes more sense that it's only the members of the island that are
![2022-05-15_21 24 29](https://user-images.githubusercontent.com/100957562/168490457-6c79fea9-ffd5-49ee-b100-521bd896552b.png)
![2022-05-15_21 24 18](https://user-images.githubusercontent.com/100957562/168490461-afab4ba0-1231-432f-bfba-bf94174f20b1.png)

**Comment by OmerBenGera** [comment, marked as answer]
If a leader wants to do that, they can.
Thanks for the suggestion tho
